packages/devtools_app/lib/src/primitives/feature_flags.dart

```diff
@@ -18,7 +18,9 @@ import '../shared/globals.dart';
 ///   ]
 const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments');
 
+@visibleForTesting
```

Comment:
since this isn't hidden is `visibleForTesting` necessary here?

Reply:
visibleForTesting protects public members from use in non-test environment: https://api.flutter.dev/flutter/meta/visibleForTesting-constant.html

It cannot make public members private.